<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>patientInfo</class>
 <widget class="QWidget" name="patientInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_4">
     <property name="minimumSize">
      <size>
       <width>1920</width>
       <height>1200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1920</width>
       <height>1200</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>731</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>110</y>
        <width>241</width>
        <height>161</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditName">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>姓名：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditGender"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>年龄：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>性别：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditAge"/>
       </item>
      </layout>
      <zorder>lineEditName</zorder>
      <zorder>lineEditAge</zorder>
      <zorder>lineEditGender</zorder>
      <zorder>label_4</zorder>
      <zorder>label</zorder>
      <zorder>label_3</zorder>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>460</y>
        <width>781</width>
        <height>331</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>0</y>
         <width>151</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;病人编号：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditIndex">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>10</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btnPrint">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>180</y>
         <width>181</width>
         <height>40</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>查看病人报表</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnEcg">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>181</width>
         <height>40</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>查询心跳</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSpO2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>140</y>
         <width>181</width>
         <height>40</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>查询血氧饱和度</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnPR">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>200</y>
         <width>181</width>
         <height>40</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>查询血压</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnHR">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>260</y>
         <width>181</width>
         <height>40</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>查询心率</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnClear">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>80</y>
         <width>181</width>
         <height>40</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>清空文本框</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnDelete">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>140</y>
         <width>181</width>
         <height>40</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>删除病人信息</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnQuery">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>200</y>
         <width>181</width>
         <height>40</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>查询病人信息</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnUpdate">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>260</y>
         <width>181</width>
         <height>40</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>编辑病人信息</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>0</y>
         <width>171</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;开始时间：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>1030</x>
         <y>120</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>查询时间</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>120</y>
         <width>141</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;间隔：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>70</y>
         <width>171</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;结束时间：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QDateTimeEdit" name="dateTimeEditBegin">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>20</y>
         <width>194</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateTimeEdit" name="dateTimeEditEnd">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>90</y>
         <width>194</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnEcgDiagram">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>260</y>
         <width>181</width>
         <height>40</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>显示心电图表</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditInterval">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>140</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>110</y>
        <width>291</width>
        <height>161</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="QPushButton" name="btnInsert">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>插入病人信息</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="btnSelectALL">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>显示全部信息</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>701</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt; font-weight:600; color:#0055ff;&quot;&gt;病人信息监测与管理平台&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>连接</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
